integration-cli/docker_cli_ps_test.go

```diff
@@ -539,3 +539,18 @@ func (s *DockerSuite) TestPsFormatMultiNames(c *check.C) {
 	}
 
 }
+
+func (s *DockerSuite) TestPsFormatHeaders(c *check.C) {
+	// make sure no-container "docker ps" still prints the header row
+	out, _ := dockerCmd(c, "ps", "--format", "table {{.ID}}")
```

Comment:
just curious, why does "table {{.ID}" result in one column but "table {{.ID}}" result in two?